### ISSUE TITLE
When Ajax submission is enabled on a multi-page form, modal shortcode page 2 does not work.

### ISSUE THREAD

**Issue by altoin** [Strategy11/formidable-modal#10]
Steps to replicate:

1. Create a multipage form
2. On page one add a modal shortcode [frmmodal-content label="Click here"]Hello Popup[/frmmodal-content]
3. Add a page break and add the same shortcode to page 2.
4. Enable Ajax submission on the form
5. Preview the page. 

The first popup will open, but the second popup will not. disable ajax submission and try this again. The popup will open on both pages.

**Comment by Crabcyborg** [comment]
@altoin I tried testing this, but it works for me.

Here's a recording.
![pvkuBkleNH](https://github.com/Strategy11/formidable-modal/assets/9134515/4990691e-a035-4466-86fb-09f2ec280642)

Here's a form on a QA site
https://qa.formidableforms.com/mike2/wp-admin/admin-ajax.php?action=frm_forms_preview&form=uebsk

Are you still able to replicate this or can this be closed?

**Comment by Crabcyborg** [comment]
@altoin You might have missed the first comment.

Are you still able to replicate this bug?

**Comment by altoin** [comment]
Hi @Crabcyborg I apologize, I missed the last comment. 

yes, I can confirm it is working now https://qa.formidableforms.com/toyin/testbug/

**Comment by Crabcyborg** [comment]
Thanks @altoin!
